docs/specifications/tests/Nameserver-TP/nameserver11.md

```diff
@@ -0,0 +1,84 @@
+# NAMESERVER11: Test for undefined EDNS OPTION-CODE
+
+## Test case identifier
+
+**NAMESERVER11** 
+
+
+## Objective
+
+EDNS is a mechanism to announce capabilities of a DNS implementation,
+and is now basically required by any new functionality in DNS such as
+DNSSEC ([RFC 6891]).
+
+[RFC 6891, section 6.1.2] states that any OPTION-CODE values not understood by a
+responder or requestor MUST be ignored. Unknown OPTION-CODE values must be
+processed as though the OPTION-CODE was not even there.
```

Comment:
In the one-line description we call it "undefined", here unknown. I suggest that we use the terminology used in the IANA repository. There we have *Reserved* (0, 4, 65535), (reserved for) *Private Use* (65001-65534), *Unassigned* (15-26945, 26947-65000). We could call everything else for "valid"?

I also think we should have the reference to the IANA repository here too.

Reply:
Following "https://datatracker.ietf.org/doc/draft-ietf-dnsop-no-response-issue/?include_text=1" i will go for "unknown"

Reply:
I think we should consider having a reference to the draft, not just in this Test Case, but in all relevant dito. Drafts are just work-in-progress, but hopefully it will end up in an RFC.

Reply:
The draft is going major modifications. I will not have a reference at this time, until it has become an RFC.